src/decider/agent/copyrightDeactivationClutterRemovalScript.py

```diff
@@ -0,0 +1,187 @@
+                list_of_copyrights = list_of_copyrights.replace(substring, "copyright")
+            
+            elif cp_symbol in list_of_copyrights:
+                list_of_copyrights = list_of_copyrights.replace(cp_symbol, "copyright")
+                
+        if substring in list_of_copyrights:
+            list_of_copyrights = list_of_copyrights.replace(substring, "copyright")
+
+        elif cp_symbol in list_of_copyrights:
+            list_of_copyrights = list_of_copyrights.replace(cp_symbol, "copyright")
+    
+        ## Implementing NER and POS Tags after normalization
+        doc2 = nlp(list_of_copyrights)
+
+        ## All the NER taggings will be contained in a dictionary having "Entity" and "Values" as keys
+        ent_dict = {}
+
+        full_table_ner = { "Entity": [], "Values": []}
+
+        for x in doc2.ents:
+            ent_dict[x.text] = x.label_
+
+        for key in ent_dict:
+            full_table_ner["Entity"].append(key)
+            full_table_ner["Values"].append(ent_dict[key])
+        
+        ## All the POS taggings will be contained in a dictionary having "Entity" and "POS_TAGS" as keys
+        pos_dict = {}
+        full_table_pos = { "Entity": [], "POS_TAG": []}
+
+        for token in doc:
+            if not token.is_punct | token.is_space:
+                pos_dict[token.text] = token.pos_
+
+        for key in pos_dict:
+            full_table_pos["Entity"].append(key)
+            full_table_pos["POS_TAG"].append(pos_dict[key])
+
+        ## The checking function call happening with each iteration
+        entityCheck(full_table_pos, full_table_ner, index, clutter_flag, df)
+    print(df.to_json(orient='records'))
+    
+
+def entityCheck(listA, listB, index, clutter_flag, df):
+    if set(THE_PROBABLE_LOGIC_POS_CHECK).intersection(set(listA["POS_TAG"])):
+        if set(THE_PROBABLE_LOGIC_NER_CHECK).intersection(set(listB["Values"])):
+                for _values in listB["Values"]:
+                    if THE_PROBABLE_LOGIC_NER_CHECK[0] in _values:
+                        df.loc[index,'is_copyright'] = "t"
+                    elif THE_PROBABLE_LOGIC_NER_CHECK[2] in _values:
+                        for _val in listB["Entity"]:
+                                pattern_regex = r'''((?:19|20)\d{2}|\d{2})(?!\d)(?:[, \t-]{1,3}((?:19|20)\d{2}|\d{1,2}))?'''
+                                extract_list = re.search(pattern_regex,_val)
+                                if extract_list:
+                                    df.loc[index,'is_copyright'] = "t"
+                    elif THE_PROBABLE_LOGIC_NER_CHECK[1] in _values or THE_PROBABLE_LOGIC_NER_CHECK[3] in _values:
+                        if THE_PROBABLE_LOGIC_NER_CHECK[0] in _values:
+                            df.loc[index,'is_copyright'] = "t"
+        else:
+            df.loc[index,'is_copyright'] = "f"
+        
+        if clutter_flag == 1:
+            if df.loc[index,'is_copyright'] == "t":
```

Comment:
This can be combined as well.
```suggestion
        if clutter_flag == 1 and df.loc[index,'is_copyright'] == "t":
```